### PR TITLE
improve wallet ui when network issues

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- feature: show/hide balance on wallet ui
- show a warning when having network issues in wallet ui
- when the network down, and back to normal => refresh the page to prevent stale data

### CODE REVIEW

File: src/pages/App.tsx [hunk 1]
```diff
@@ -108,6 +109,16 @@ const BodyWrapper = styled.div`
 export default function App() {
   const { account, chainId, networkInfo } = useActiveWeb3React()
 
+  const { online } = useNetwork()
+  const prevOnline = usePrevious(online)
```
Comment (hungdoansy) [inline src/pages/App.tsx:113]: usePrevious seems to have a `previous` field ([ref](https://github.com/streamich/react-use/blob/master/src/useNetworkState.ts#L91)). Can we use it?

File: src/components/WalletPopup/MyAssets.tsx [hunk 2]
```diff
@@ -40,11 +40,15 @@ export default function MyAssets({
   loadingTokens,
   usdBalances,
   currencyBalances,
+  isNetworkIssue,
+  hideBalance,
 }: {
   tokens: Currency[]
   loadingTokens: boolean
+  isNetworkIssue: boolean
```
Comment (hungdoansy) [inline src/components/WalletPopup/MyAssets.tsx:48]: this should be `hasNetworkIssue`

File: src/pages/App.tsx (hunk 1 above)
Comment (nguyenhoaidanh) [reply src/pages/App.tsx:113]: no, maybe we use different version
image

File: src/components/WalletPopup/MyAssets.tsx (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/components/WalletPopup/MyAssets.tsx]: updated. continue review pls 🚀

**Pull request merged**